changelog/gc_suffixes.dd

```diff
@@ -0,0 +1,12 @@
+Memory-related GC options stored in bytes
```

Comment:
Storage is not a user visible change, I'd rather describe it as "Memory releated GC options can now be specified with higher accuracy" or similar.